src/pages/background/services/signify.ts

```diff
@@ -150,6 +150,11 @@ const Signify = () => {
     return signed_headers;
   };
 
+  const getAgentID = (): string => {
```

Comment:
Consider adding a call for getControllerID that accesses _client?.controller?.pre!